app/controllers/developers_controller.rb

```diff
@@ -44,12 +46,24 @@ def filter
 private
 
   def sort
-    { 
+    {
       "Rate: Ascending" => lambda { |active| active.order(rate: :asc) },
       "Rate: Descending" => lambda { |active| active.order(rate: :desc) }
     }
   end
 
+  def selection_filter(initial_developers)
+    preload = initial_developers.joins(:tags)
+    intersection_query = selected_tags.reduce(initial_developers) do |acc, tag_name|
```

Comment:
Useless assignment to variable - `intersection_query`.<br>Line is too long. [84/80]<br>Prefer `inject` over `reduce`.